handler.py

```diff
@@ -14,38 +16,120 @@
 SQS_CLIENT = boto3.client('sqs')
 
 
-def runScheduledObservatoryScan(event, context):
+def addPortScanToQueue(event, context):
+    data = json.loads(event['body'])
+    target = Target(data.get('target'))
```

Comment:
What happens when the data structure does not contain the 'target' element?  Does it produce a None Object or empty string?  I think this is important if we haven't hardened target to handle alt Object types beyond str.

Reply:
target becomes `null`, and fails the `isValid` function. Can probably handle this a step before the `isValid` function.